bach/bach/series/series_datetime.py

```diff
@@ -313,12 +318,13 @@ class SeriesTimedelta(SeriesAbstractDateTime):
     supported_value_types = (datetime.timedelta, numpy.timedelta64, str)
 
     @classmethod
-    def supported_literal_to_expression(cls, literal: Expression) -> Expression:
+    def supported_literal_to_expression(cls, dialect: Dialect, literal: Expression) -> Expression:
         return Expression.construct('cast({} as interval)', literal)
```

Comment:
why not? I'm aware SeriesTimedelta is supported only for Postgres, but IMHO we should still get the db type from the mapping.
```suggestion
        db_dialect = DBDialect.from_dialect(dialect)
        db_dtype = cls.supported_db_dtype[db_dialect]
        return Expression.construct(f'cast({{}} as {db_dtype})', literal)
```

Reply:
Changed in 904c7a5f330f9d59c1ec1a849d48b9040ffdd945